addon/components/notification-message.js

```diff
@@ -1,37 +1,58 @@
 import Ember from 'ember';
 
 export default Ember.Component.extend({
-    classNameBindings: [':c-notification', 'processedType', 'notification.dismiss::c-notification--in', 'autoClear::c-notification--dismissable'],
+  classNames: ['c-notification'],
+  classNameBindings: [
+    'processedType',
+    'notification.dismiss::c-notification--in',
+    'autoClear::c-notification--dismissable'
+  ],
 
-    // Set icon depending on notification type
-    notificationIcon: Ember.computed('notification.type', function() {
-        switch(this.get('notification.type')){
-            case "info":
-                return 'fa-info-circle';
-            case "success":
-                return 'fa-check';
-            case "warning":
-                return 'fa-warning';
-            case "error":
-                return 'fa-exclamation-circle';
-        }
-    }),
+  // Set icon depending on notification type
+  notificationIcon: Ember.computed('notification.type', 'theme', function() {
+    var icons = this.get('icons');
 
-    processedType: Ember.computed('notification.type', function() {
-      if(this.get('notification.type') && Ember.A(['info', 'success', 'warning', 'error']).contains(this.get('notification.type'))){
-        return "c-notification--" + this.get('notification.type');
+    if (icons === 'font-awesome') {
+      switch(this.get('notification.type')){
+        case "info":
+          return 'fa fa-info-circle';
+        case "success":
+          return 'fa fa-check';
+        case "warning":
+          return 'fa fa-warning';
+        case "error":
```

Comment:
This isn't about a default icon style (font awesome or bootstrap) it's about having a default icon for when you don't specify info success warning or error